### ISSUE TITLE
Review Build Issues (Jekyll)

### ISSUE THREAD

**Issue by budparr** [theNewDynamic/mfocus-tracking#1]
- [ ] Run Jekyll 
- [ ] Troubleshoot issues in build
https://github.com/missionfocus/www.missionfocus.com

**Comment by andreweick** [comment]
Netlify deploy error:

```
9:18:20 AM: Build ready to start
9:18:22 AM: build-image version: 8e315e54bc4032a32e73290be556cde4f8348c12
9:18:22 AM: build-image tag: v2.8.2
9:18:22 AM: buildbot version: 45cd00068410d70db72c9672cdd014995488e12a
9:18:22 AM: Fetching cached dependencies
9:18:22 AM: Failed to fetch cache, continuing with build
9:18:22 AM: Starting to prepare the repo for build
9:18:22 AM: git ref refs/heads/master does not exist or you do not have permission
9:18:22 AM: Failing build: Failed to prepare repo
9:18:22 AM: failed during stage 'preparing repo': git ref refs/heads/master does not exist
```

**Comment by budparr** [comment]
@andreweick We had a look at this and think the most efficient course of action would be to migrate the site to Hugo. We think, considering how small the site is, it wouldn't be much more effort than continuing to troubleshoot (which is a bit open-ended anyway) and would leave you in better shape for changes in the future.

**Comment by andreweick** [comment]
@budparr agree -- recommend migrating to Hugo, ignoring any hard or weird bits, and then we'll work on a redesign / rethink when I have some time

**Comment by budparr** [comment]
Closed in #6
